.devcontainer/cuda12.2-conda/devcontainer.json

```diff
@@ -0,0 +1,74 @@
+{
```

Comment:
Our `make_devcontainers.sh` script will delete `.devcontainer/` subdirs that don't exist in our generated CI matrix. You'll need to update that script so that it keeps this directory when `--clean` is passed as an argument.

Ideally, it should also parse the required information from the workflow files and automatically generate this file so that the devcontainer stays up-to-date, but I'm not sure how feasible that would be.

Reply:
Yeah... I guess I could special-case it in that script?

Alternatively, I could place this devcontainer somewhere other than the `.devcontainer` folder, but then VSCode wouldn't include it in the list of devcontainers provided to users. A user could still modify things locally to get it to launch if they wanted to debug CCCL and RAPIDS, but it would be an extra step a developer would need to know to do.

Reply:
> VSCode wouldn't include it in the list of devcontainers provided to users

I think this is actually want I would prefer. These devcontainers are very specific to RAPIDS stuff. I don't want people getting confused and thinking these are the same as the rest of our devcontainers.